### PR TITLE
feat(client): esm support

### LINKED ISSUE #5030: Support ES6 modules export
It would be great if the Prisma Client JS generator would be compatible with ES6 modules out of the box.

Note: Related to prisma/prisma-client-js#175

### PR DESCRIPTION
**Pull request by timsuchanek**
Continuing https://github.com/prisma/prisma/pull/4178
Closes prisma/prisma-client-js#539

### CODE REVIEW

**Comment by timsuchanek** [comment]
@muxahuk I tried to integrate your PR here, but it looks like it would be a bit more work.
The important special requirement for Prisma Client that you need to keep in mind:
Prisma Client bundles everything and has not dependencies.
That's why we use `esbuild` and `rollup` to both bundle the JavaScript and TypeScript.
Feel free to continue in this branch or merge this one into your original one.
I need to work on other things now, but I'm happy to review anything you work on 🙏 .

**Comment by muxahuk** [comment]
Hey @timsuchanek i had some issues pushing to this branch, so i pushed to my https://github.com/prisma/prisma/pull/4178 some tests seem to be passing fine there..

**Comment by anonrig** [comment]
hey all, are there any progress on this task, and if not, do you need any help to pass this pull request?

**Comment by timsuchanek** [comment]
Hey @anonrig! I started working on this again, so soon this will probably be merged :)

File: src/packages/client/package.json [hunk 1]
```diff
@@ -97,6 +110,7 @@
     "make-dir": "3.1.0",
     "mariadb": "2.5.2",
     "mssql": "6.3.1",
+    "ncp": "^2.0.0",
```
Comment (Jolg42) [inline src/packages/client/package.json:113]: We already have `@timsuchanek/copy` in this package, I guess we don't need `ncp` https://www.npmjs.com/package/ncp

**Comment by timsuchanek** [comment]
Closing in favor of https://github.com/prisma/prisma/pull/4920

**Pull request closed without merging**